NSS-Day-Cohort-49/tabloid-full-stack-spicy-sweet-chilis
# Jad view categ

#### Changes Made
1. Added file `Category.js, CategoryList.js` to `Category` directory.
2. Added file `CategoryManager.js` to `modules` directory.
3. Added file `Category.cs, CategoryController.cs, CategoryRepository.cs, ICategoryRepository.cs to Tabloid.sln
4. edited Startup.cs, ApplicationViews.js, and Header.js

#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout jad-ViewCateg
    ```
2. Open a new Terminal tab (?T) and navigate to the server directory.
3. Test app functionality.
    > navigate using header to "Category Management" to see a list of all available categories in ABC order
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
